### PR TITLE
chore: improvements to date format howto and extra references to how to change with i3 config

### PR DESCRIPTION
**Pull request by thebetar**
After my last commit I have decided to extend the documentation wherever I found it difficult to understand. Since I am a beginnern without any experience with using i3 but with experience of basic Unix I think extending the docs with small hints I would have found useful will help other new users have a better understanding of what is being meant.

One thing I think would still be a great idea would be to include an example `xresources` file somewhere in the docs I think preferably in the `configuration.md` but since my custom configuration is not very extensive yet I think probably one of the long term users can create a simple setup that is recommended for more users.

### CODE REVIEW

File: content/docs/howtos/change-background.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+title: 'Change background'
+description: >
+    Change the background
+---
+
+# Wallpaper
+
+Most Regolith Looks provide a default desktop color or image. The wallpaper can be set by specifying the path to the wallpaper image via the Xresources key `regolith.wallpaper.file`.
```
Comment (kgilmer) [inline content/docs/howtos/change-background.md:9]: completeness: wallpaper can also be configured by the settings app `regolith-control-center`.  Can you mention that as well for users that prefer a GUI?

File: content/docs/reference/i3xrocks.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: 'i3xrocks'
+linkTitle: 'i3xrocks'
+weight: 7
+description: >
+    A small intro into what `i3xrocks` is.
+---
+
+Since i3 is the underlying tiling manager that is used by regolith, the `i3rocks` package can also be used. However since the choice was made to use `Xresources` this package has been forked and `i3xrocks` was born (created by the regolith team).
```
Comment (kgilmer) [inline content/docs/reference/i3xrocks.md:9]: correctness/nit: the upstream fork is called `i3blocks`.  https://github.com/vivien/i3blocks

**Review by kgilmer — CHANGES_REQUESTED** [review]
Fantastic work, two trivial items..

File: content/docs/howtos/change-background.md (hunk 1 above)
Comment (thebetar) [reply content/docs/howtos/change-background.md]: I have added a mention to the Regolith control center is this what you meant?

File: content/docs/reference/i3xrocks.md (hunk 2 above)
Comment (thebetar) [reply content/docs/reference/i3xrocks.md]: I have corrected it and added links to the corresponding repositories

**Comment by thebetar** [comment]
this is everything I have found up till now, I am currently busier with university but will try to add some more documentation later enxt month.

**Review by kgilmer — APPROVED** [review]

**Comment by thebetar** [comment]
It seems that this PR broke the build with the reference made in `content/docs/howtos/customize-i3-configuration.md` I have updated the reference in my last commit.

**Pull request merged**